InterviewFlashCardsTests/InterviewFlashCardsTests.m

```diff
@@ -22,7 +13,6 @@ @implementation InterviewFlashCardsTests
 
 - (void)setUp {
     [super setUp];
-    self.qman = [IFCQueryManager new];
```

Comment:
No worries, it just means if we were to merge this code to master then it will be missing/removed from master. So make sure you go through each line you commit and ensure that it should be included in your current commit/PR. I suggest using `git add -p`, to add files you changed line by line.